cli/src/cmd/utils.rs

```diff
@@ -228,3 +230,33 @@ pub fn has_batch_support(chain: u64) -> bool {
     }
     true
 }
+
+pub trait LoadConfig {
+    fn load_config_emit_warnings(self) -> Config;
+    fn load_config_and_evm_opts_emit_warnings(self) -> eyre::Result<(Config, EvmOpts)>;
+    fn load_config_unsanitized_emit_warnings(self) -> Config;
+}
```

Comment:
this needs to be documented as how this differs from regular config parsing
since this could be confusing